try: 
	raise NameError("error here") # Raise Error 
except NameError: 
	print ("An exception") 
